### PR TITLE
Add triple slash comments for new System.Drawing API added in 6.0

### PR DESCRIPTION
**Pull request by safern**
Contributes to: https://github.com/dotnet/runtime/issues/54865

### CODE REVIEW

File: src/libraries/Common/src/System/Drawing/KnownColor.cs [hunk 1]
```diff
@@ -197,6 +197,9 @@ public enum KnownColor
         MenuHighlight,
 
         // "Web" colors, Part 2
+        /// <summary>
+        /// A system defined color representing the ARGB value <c>#663399</c>.
```
Comment (stephentoub) [inline src/libraries/Common/src/System/Drawing/KnownColor.cs:201]: Looking at the existing docs, most of these seem to just be "A system-defined color.".  I don't have a strong opinion, but calling out what seems like a discrepancy (if I'm looking in the right place).

**Review by stephentoub — APPROVED** [review]

File: src/libraries/System.Drawing.Primitives/src/System/Drawing/Color.cs [hunk 2]
```diff
@@ -249,6 +249,10 @@ namespace System.Drawing
 
         public static Color Purple => new Color(KnownColor.Purple);
 
+        /// <summary>
+        /// Gets a system-defined color that has an ARGB value of <c>#663399</c>.
+        /// </summary>
+        /// <value>A <see cref="Color" /> representing a system-defined color.</value>
```
Comment (gewarren) [inline src/libraries/System.Drawing.Primitives/src/System/Drawing/Color.cs:255]:
```suggestion
        /// <value>A system-defined color.</value>
```

No need to specify the type.

**Review by gewarren — APPROVED** [review]

File: src/libraries/Common/src/System/Drawing/KnownColor.cs (hunk 1 above)
Comment (safern) [reply src/libraries/Common/src/System/Drawing/KnownColor.cs:201]: I saw that some contained the ARGB code, i.e: `Purple`. That's why I added it, but I don't have a strong opinion either.

Comment (safern) [reply src/libraries/Common/src/System/Drawing/KnownColor.cs:201]: I will merge this PR as is and if you have a strong opinion I can submit a new PR to fix that.

**Review by danmoseley — APPROVED** [review]

**Comment by safern** [comment]
Failures are unrelated.

**Pull request merged**